pkg/pit/customizations_yaml.go

```diff
@@ -205,7 +205,7 @@ func GenCustomizationsYaml(ncns []csi.LogicalNCN, shastaNetworks map[string]*csi
 			NMNSubnetCIDR:              uaiNetCIDR.String(),
 			NMNSupernetGateway:         uaiNet.Gateway,
 			NMNSupernetCIDR:            shastaNetworks["NMN"].CIDR,
-			NMNVlanInterface:           fmt.Sprintf("vlan%03d", uaiNet.VlanID),
+			NMNVlanInterface:           "bond0.nmn0",
```

Comment:
I'm not too familiar with this code, but in another file where the new interface was hardcoded, csi couldn't run on an  older 1.0 system.  Maybe unlikely in this case, but something to check if you feel like it.

Reply:
The hard-code is not ideal for sure.  It shouldn't cause problems in this case.  We're also hard-coding it for cloud-init/writefiles.  Also not ideal...

Reply:
There is a var somewhere in CSI that does determine 1.0 or 1.2, incase it's used in either context 🤔  it definitely went in not too recently.